spec/lucky/rate_limit_spec.cr

```diff
@@ -0,0 +1,43 @@
+require "../spec_helper"
+
+include ContextHelper
+
+class RateLimitRoutes::Index < TestAction
+  include Lucky::RateLimit
+
+  get "/rate_limit" do
+    plain_text "hello"
+  end
+
+  private def rate_limit : NamedTuple(to: Int32, within: Time::Span)
+    {to: 1, within: 1.minute}
+  end
```

Comment:
This is actually a pretty slick interface. It feels consistent with some other Lucky modules.